src/resolvers/get-model.js

```diff
@@ -178,6 +178,18 @@ const getModelFromMakeDescription = (make, description) => {
       }
       break
     }
+    case Make.POLESTAR: {
+      if (description.match(/1$/i)) {
+        return Model[make][1]
+      }
+      if (description.match(/2$/i)) {
+        return Model[make][2]
+      }
+      if (description.match(/3$/i)) {
+        return Model[make][3]
+      }
+      break
+    }
```

Comment:
Here as well with `POL...` being before `POR...`.

Reply:
I'm also not sure about these matchers. It doesn't seem unlikely that someone would add something behind the "Polestar 2" text in the description. And this would match "Polestar 2 HP 233" as a Polestar 3.

If you're matching just the numbers, I would match a space or `^` in front as well. And I would probably also allow matching of `/polestar 2/i`.

Reply:
I've added a space and allowed matching on `/polestar \d/i` now